src/components/Welcome.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Welcome({ items }) {
+  //if(!items){
+  return (
+    <div>
+      <h1 className="font-Karma text-black text-center text-4xl my-4">
+        Welcome to ~name of app here~
+      </h1>
+
+      <div className="rounded-xl w-1/2 mx-auto  mt-10 bg-gradient-to-r p-4 from-orange-500 via-yellow-500 to-orange-500">
```

Comment:
@djtaylor8 @lindseyindev 
Wow how fun - I haven't used tailwind before so I'm enjoying learning about it through you all! 

Do either of you know if there's a way to make the class names a little more readable? (Thinking particularly for the line that you need to scroll for).

Reply:
Oh yes the readability of tailwind CSS... I'd recommend adding comments above about what that style row does, for example:
<-- Style centers the block with blue background, and has 3 children columns on large or 2 on medium (for mobile) -->

Reply:
> @djtaylor8 @lindseyindev Wow how fun - I haven't used tailwind before so I'm enjoying learning about it through you all!
> 
> Do either of you know if there's a way to make the class names a little more readable? (Thinking particularly for the line that you need to scroll for).

We can look into this. It's definitely one of the primary complaints against Tailwind - the class names can get unwieldy pretty quickly.